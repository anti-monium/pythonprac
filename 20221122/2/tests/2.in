ïàðîâîç ïî ðåëüñàì ì÷èòñÿ,
íà ïóòè êîò¸íîê ñïèò.
ïàðîâîç îñòâíîâèëñÿ
è êîò¸íêó ãîâîðèò:
"òû êîò¸íîê, òû êîò¸íîê,
îòîéäè êóäà íèáóäü!",
à êîò¸íîê îòâå÷àåò:
"ñàì ïðîåäåøü êàê íèáóäü!".
ïàðîâîçèê ðàññåðäèëñÿ -
îòäàâèë êîò¸íêó õâîñò;
à êîò¸íîê ðàññåðäèëñÿ -
îöàðàïàë ïàðàâîç!
ìàìà êîøêà ó îêîøêà
ïðèøèâàåò ñûíó õâîñò
è êîò¸íî÷êà ðóãàåò,
÷òîá íå ëåç ïîä ïàðîâîç.
ïàðîâîç ëåæèò â áîëüíèöå - 
åìó äåëàþò óêîë,
à êîò¸íîê ñïèò íà ïå÷êå
ñ ïåðåâÿçàííûì õâîñòîì

