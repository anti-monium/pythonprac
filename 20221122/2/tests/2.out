паровоз по рельсам мчится,
на пути котёнок спит.
паровоз оствновился
и котёнку говорит:
"ты котёнок, ты котёнок,
отойди куда нибудь!",
а котёнок отвечает:
"сам проедешь как нибудь!".
паровозик рассердился -
отдавил котёнку хвост;
а котёнок рассердился -
оцарапал паравоз!
мама кошка у окошка
пришивает сыну хвост
и котёночка ругает,
чтоб не лез под паровоз.
паровоз лежит в больнице - 
ему делают укол,
а котёнок спит на печке
с перевязанным хвостом
